traffic_ops/testing/api/v3/servers_test.go

```diff
@@ -358,6 +359,39 @@ func GetTestServersQueryParameters(t *testing.T) {
 	if !foundTopDs {
 		t.Fatalf("unable to find deliveryservice %s", topDsXmlId)
 	}
+
+	/* Create a deliveryservice server assignment that should not show up in the
+	 * client.GetServersWithHdr() response because ds-top is topology-based
+	 */
+	const otherServerHostname = "topology-edge-02"
+	serverResponse, _, err := TOSession.GetServersWithHdr(&url.Values{"hostName": []string{otherServerHostname}}, nil)
+	if err != nil {
+		t.Fatalf("getting server by hostname %s: %s", otherServerHostname, err)
+	}
+	if len(serverResponse.Response) != 1 {
+		t.Fatalf("unable to find server with hostname %s", otherServerHostname)
+	}
+	otherServer := serverResponse.Response[0]
+
+	db, err := OpenConnection()
```

Comment:
Should we use the TO Go client to do this instead of making a direct DB query? `CreateDeliveryServiceServers()` or `AssignServersToDeliveryService()`?

Reply:
We should, I was just avoiding our API validation on DS server assignments. Using the TO Go client library in 3851d4abdc.